lib/src/base/currency_cloud_base.dart

```diff
@@ -12,15 +12,21 @@ abstract class CurrencyCloudApi {
 /// [CurrencyCloudClient] is used for communication with the CurrencyCloudService. It provides Request methods
 /// like [get] and [post] which handle the basic communication overheads like adding authentication headers.
 class CurrencyCloudClient {
-  static final String baseUri = 'https://devapi.thecurrencycloud.com/v2';
+  static final String devUri = 'https://devapi.thecurrencycloud.com/v2';
+  static final String liveUri = 'https://api.thecurrencycloud.com/v2';
+
+  String baseUri;
 
   String loginId, apiKey;
 
   AuthToken _authToken;
   bool get isAuthenticated => _authToken.isSet;
 
-  CurrencyCloudClient(this.loginId, this.apiKey) {
+  /// Provide true for [useLiveUri] if you want to use this client in production and trigger real money transfers
+  CurrencyCloudClient(this.loginId, this.apiKey, {bool useLiveUri: false}) {
```

Comment:
`CurrencyCloudClient(this.loginId, this.apiKey, {bool useLiveUri: false}) : baseUri = useLiveUri ? liveUri : devUri;`